Configuration/TypoScript/Base/Page.txt

```diff
@@ -0,0 +1,101 @@
+page = PAGE
+page {
+
+    10 = FLUIDTEMPLATE
+    10 {
+        extbase {
+            pluginName = PageRenderer
+            controllerExtensionName = KoningTemplate
+            controllerVendorName = KoninklijkeCollective
+            controllerName = Page
+            controllerActionName = render
+        }
+
+        templateRootPaths {
+            10 = EXT:koning_template/Resources/Private/Templates/
+        }
+        partialRootPaths {
+            10 = EXT:koning_template/Resources/Private/Partials/
+        }
+        layoutRootPaths {
+            10 = EXT:koning_template/Resources/Private/Layouts/
+        }
+
+        templateName.cObject = CASE
+        templateName.cObject {
+            key.data = levelfield:-1, backend_layout_next_level, slide
+            key.override.field = backend_layout
+
+            {$layout.default} = TEXT
+            {$layout.default}.value = Default
+
+            default = TEXT
+            default.value = Unknown
+        }
+
+        settings {
+            pages {
+                root = {$pages.root}
+            }
+        }
+    }
+
+    meta {
+        keywords.field = keywords
+        description.field = description
+        X-UA-Compatible = IE=edge,chrome=1
+        X-UA-Compatible.httpEquivalent = 1
+        viewport = width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no
+    }
+
+    includeJSFooter {
+        grunt-compiled = EXT:koning_template/Resources/Public/JavaScript/Footer.js
+    }
+
+    includeCSSFooter {
+        grunt-compiled = EXT:koning_template/Resources/Public/Css/Compiled.css
+    }
+
+    headerData {
+        10 = TEXT
+        10 {
+            typolink {
+                parameter.data = TSFE:id
+
+                addQueryString = 1
+
+                type = 0
+
+                forceAbsoluteUrl = 1
+                forceAbsoluteUrl.scheme = https
+                returnLast = url
+            }
+
+            wrap = <link rel="canonical" href="|" />
+        }
+
+        20 = USER
+        20.userFunc = KoninklijkeCollective\KoningTemplate\UserFunc\PageRenderer->criticalCss
+        20.file = EXT:koning_template/Resources/Public/Css/Critical.css
+    }
+
+    footerData {
```

Comment:
@bmoex, weet jij een betere manier om css in de footer te krijgen? includeCSSFooter werkt niet :(